templates/workloads/scheduled-job/state-machine-definition.json

```
@@ -0,0 +1,58 @@
+{
+  "Version": "1.0",
+  "Comment": "Run AWS Fargate task",{{if .Timeout}}
+  "TimeoutSeconds": ${Timeout},{{end}}
```

nit: to make our go templates easy to read we can leverage the minus sign:
> However, to aid in formatting template source code, if an action's left delimiter (by default "{{") is followed immediately by a minus sign and ASCII space character ("{{- "), all trailing white space is trimmed from the immediately preceding text. Similarly, if the right delimiter ("}}") is preceded by a space and minus sign (" -}}"), all leading white space is trimmed from the immediately following text.
https://golang.org/pkg/text/template/

```suggestion
  "Comment": "Run AWS Fargate task",
  {{- if .Timeout}}
  "TimeoutSeconds": ${Timeout},
  {{- end}}
```